environment.yml

```diff
@@ -18,5 +18,5 @@ dependencies:
 - joblib
 - memory_profiler
 - pip:
-  - mne
+  - mne==0.23.0
```

Comment:
should be possible to remove this now after rebase @cedricrommel

Reply:
+1